lntest/node/harness_node.go

```diff
@@ -97,7 +97,12 @@ type HarnessNode struct {
 func NewHarnessNode(t *testing.T, cfg *BaseNodeConfig) (*HarnessNode, error) {
 	if cfg.BaseDir == "" {
 		var err error
-		cfg.BaseDir, err = ioutil.TempDir("", "lndtest-node")
+
+		// Create a temporary directory for the node's data and logs.
+		// Use dash suffix as a separator between base name and random
+		// suffix.
+		dirBaseName := fmt.Sprintf("lndtest-node-%s-", cfg.Name)
+		cfg.BaseDir, err = os.MkdirTemp("", dirBaseName)
```

Comment:
isn't this temp dir being removed after the test finishes?

Reply:
Either the answer is no, or I think there are options in place to disable that automatic cleanup.

Reply:
Or these directories persist whilst my breakpoint is in place.